core/src/main/java/com/web3auth/core/Web3Auth.kt

```diff
@@ -438,7 +440,7 @@ class Web3Auth(web3AuthOptions: Web3AuthOptions) {
     private fun getLoginId(jsonObject: JSONObject): CompletableFuture<String> {
         val createSessionCompletableFuture: CompletableFuture<String> = CompletableFuture()
         val sessionResponse: CompletableFuture<String> =
-            sessionManager.createSession(jsonObject.toString(), 600, false)
+            sessionManager.createSession(jsonObject.toString(), 600, web3AuthOption.context)
```

Comment:
Pass this context in as a parameter to the function, remove it from web3authoption.

Context should not be held unnecessarily since it can lead to memory leaks.

Reply:
There is more than one place this needs to be changed